### PR TITLE
Integration test

### PR DESCRIPTION
**Pull request by khannaabhi**

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -2,128 +2,50 @@ cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMT
 cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 cloud.google.com/go v0.38.0 h1:ROfEUZz+Gh5pa62DJWXSaonyu3StP6EA6lPEXPI6mCo=
 cloud.google.com/go v0.38.0/go.mod h1:990N+gfupTy94rShfmMCWGDn0LpTmnzTp2qbd1dvSRU=
-github.com/Azure/azure-sdk-for-go v16.0.0+incompatible h1:gr1qKY/Ll72VjFTZmaBwRK1yQHAxCnV25ekOKroc9ws=
```
Comment (sharadregoti) [inline go.sum]: @YourTechBud what are these changes ?

File: modules/schema/creation_test.go [hunk 2]
```diff
@@ -6,148 +6,1379 @@ import (
...
+			args: args{
+				dbAlias:       "postgres",
+				tableName:     "table1",
+				project:       "test",
+				parsedSchema:  schemaType{"postgres": schemaCollection{"table1": SchemaFields{"col1": &SchemaFieldType{FieldName: "col1", Kind: TypeID, IsFieldTypeRequired: false, IsForeign: false}}, "table2": SchemaFields{"id": &SchemaFieldType{FieldName: "col1", Kind: TypeID}}}},
+				currentSchema: schemaCollection{"table1": SchemaFields{"col1": &SchemaFieldType{FieldName: "col1", Kind: TypeID, IsForeign: true, JointTable: &TableProperties{Table: "table2", To: "id"}}}},
 			},
-			Enabled: true,
+			fields:  fields{crud: crudPostgres, project: "test"},
+			want:    []string{"ALTE
```
Comment (sharadregoti) [inline modules/schema/creation_test.go]: @YourTechBud does the return statement here if an error occurs will stop testing the other test cases?

File: go.sum (hunk 1 above)
Comment (YourTechBud) [reply go.sum]: let it be

File: modules/schema/creation_test.go (hunk 2 above)
Comment (YourTechBud) [reply modules/schema/creation_test.go]: There should be no return statements here

Comment (sharadregoti) [inline modules/schema/creation_test.go]: @khannaabhi remove the return statement

**Pull request merged**